admin_guide/upgrading_citus.rst

```diff
@@ -49,15 +49,15 @@ Step 1. Update Citus Package
 .. code-block:: bash
 
   sudo apt-get update
-  sudo apt-get install postgresql-9.6-citus-7.1
+  sudo apt-get install postgresql-9.6-citus-7.2
   sudo service postgresql restart
 
 **Fedora, CentOS, or Red Hat**
 
 .. code-block:: bash
 
   # Fedora, CentOS, or Red Hat
-  sudo yum swap citus62_96 citus71_96
+  sudo yum swap citus62_96 citus72_96
```

Comment:
I better try this line out one more time because of the tricky dependency in `citus72_96`.

Reply:
OK, I installed citus62_96 by modifying our script, and once that was there the `yum swap` worked fine.